docker/lr-pb/collect_zmw_subread_stats.py

```diff
@@ -0,0 +1,166 @@
+import pysam
```

Comment:
same assumption

Reply:
Correct - this was tested a long time ago with a few different bam files.